com.unicornsonlsd.Finamp.yaml

```diff
@@ -0,0 +1,291 @@
+app-id: com.unicornsonlsd.Finamp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: finamp
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.*
+modules:
+  - name: Finamp
+    buildsystem: simple
+    build-commands:
+      - cp -r finamp /app/finamp
+      - chmod +x /app/finamp/finamp
+      - install -Dm644 icon_foreground.svg /app/share/icons/hicolor/scalable/apps/com.unicornsonlsd.Finamp.svg
+      - mkdir -p /app/bin
+      - ln -s /app/finamp/finamp /app/bin/finamp
+      - install -Dm644 com.unicornsonlsd.Finamp.desktop /app/share/applications/com.unicornsonlsd.Finamp.desktop
+      - install -Dm644 com.unicornsonlsd.Finamp.metainfo.xml /app/share/appdata/com.unicornsonlsd.Finamp.metainfo.xml
+    sources:
+      - type: archive
+        dest: finamp
+        url: https://github.com/jmshrv/finamp/releases/download/0.9.7-beta/finamp-0.9.7-beta-linux-release.tar.gz
```

Comment:
how does that work on aarch64? Why not building from source given the license?

Reply:
It doesn't, I'll need to add the arches (a dependency currently doesn't build on aarch64 so only an x86 build is possible at the moment). My understanding was is it's not currently possible to build flutter apps from source because of Flathub's requirement for offline builds but if that's wrong I'll have a look at doing that.

Reply:
Added the `x86_64` constraint.

Reply:
> Added the `x86_64` constraint.

You need https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on too before we can build.